src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp

```diff
@@ -680,6 +699,16 @@ ConvSolution ConvHipImplicitGemmBwdDataV4R1Xdlops::GetSolution(
                      GemmBBlockCopyDstDataPerWrite_GemmN,
                      std::ignore) = config.CalculateGemmBBlockCopyPerformanceParameters(ctx);
 
+            GemmABlockCopyDstDataPerWrite_GemmM =
+                ctx.IsFp32() ? GemmABlockCopyDstDataPerWrite_GemmM : 1;
+            GemmBBlockCopyDstDataPerWrite_GemmN =
+                ctx.IsFp32() ? GemmBBlockCopyDstDataPerWrite_GemmN : 1;
+
+            const auto GemmABlockCopyDstDataPerWrite_GemmKPACK =
+                !ctx.IsFp32() ? GetEPackLength(ctx, true) : 1;
```

Comment:
Please make EPack to be tunable.  We have made EPack tunable in fwd bfp16 case as it will try 2 and 4. Though not for fp16.

Here is code examples:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/3ed5e8569ec82ea2d5f8706d279e2074ceeb8a5a/src/solver/conv_hip_implicit_gemm_v4r4_gen_xdlops.cpp#L190
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/3ed5e8569ec82ea2d5f8706d279e2074ceeb8a5a/src/solver/conv_hip_implicit_gemm_xdlops_common.cpp#L268

https://github.com/ROCmSoftwarePlatform/MIOpen/blob/3ed5e8569ec82ea2d5f8706d279e2074ceeb8a5a/src/solver/conv_hip_implicit_gemm_xdlops_common.cpp#L55

Reply:
Ok. I understand. I will make it tunable.

Reply:
@TejashShah It is tunable for bfp16